src/index.html

```diff
@@ -6,9 +6,98 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+         src="./images/logo.png"
+         alt="company logo"
+        >
+      </div>
```

Comment:
This element should be the link to the home page.
![image](https://user-images.githubusercontent.com/60885495/159488860-d22798cb-301e-492d-9d16-f3239b8993fa.png)